update-test-group.sh

```diff
@@ -1,20 +1,22 @@
 #!/usr/bin/env bash
 # This script sets a group of devices to a specific commit,
-# currently it will look for devices in an app that have an environment variable called "TEST"
-# these devices will then update to whatever commit is supplied as arg $1
+# currently it will look for devices in an app that have a tag specified as arg $1
+# these devices will then update to whatever commit is supplied as arg $2
 
 ./check-configuration.sh || exit 1
 
-COMMIT_HASH=$1
+TARGET_TAG_KEY=$1
```

Comment:
* We should probably update the README as well to reflect how this now works.
* Just-an-idea: how about expecting the commit hash as a first argument and have the `TARGET_TAG_KEY` default to `TEST` if it's not provided?

Reply:
Good idea, I did this (which required inverting the order of arguments)